frontend/app/src/components/dashboard/LiquidityProviderBalanceTable.vue

```diff
@@ -175,13 +195,68 @@ const createTableHeaders = (
 const route = Routes.DEFI_DEPOSITS_LIQUIDITY.route;
 const expanded = ref<XswapBalance[]>([]);
 
-const { uniswapV3Balances } = useUniswapStore();
+const {
+  uniswapV2Balances,
+  uniswapV3Balances,
+  fetchV2Balances: fetchUniswapV2Balances,
+  fetchV3Balances: fetchUniswapV3Balances
+} = useUniswapStore();
+
+const {
+  balanceList: sushiswapBalances,
+  fetchBalances: fetchSushiswapBalances
+} = useSushiswapStore();
+
+const balancerStore = useBalancerStore();
+
+const { balanceList: balancerBalances } = storeToRefs(balancerStore);
+const { fetchBalances: fetchBalancerBalances } = balancerStore;
+
 const balances = computed(() => {
-  return get(uniswapV3Balances([])).map(item => ({
+  const mappedUniswapV3Balances = get(uniswapV3Balances([])).map(item => ({
+    ...item,
+    usdValue: item.userBalance.usdValue,
+    asset: item.nftId,
+    premiumOnly: true,
+    type: 'nft'
+  }));
+
+  const mappedUniswapV2Balances = get(uniswapV2Balances([])).map(item => ({
     ...item,
     usdValue: item.userBalance.usdValue,
-    name: item.nftId ? getNftBalance(item.nftId)?.name ?? item.nftId : ''
+    asset: `_ceth_${item.address}`,
+    premiumOnly: false,
+    type: 'token'
+  }));
+
+  const mappedSushiswapBalances = get(sushiswapBalances([])).map(item => ({
+    ...item,
+    usdValue: item.userBalance.usdValue,
+    asset: `_ceth_${item.address}`,
+    premiumOnly: true,
+    type: 'token'
+  }));
+
+  const mappedBalancerBalances = get(balancerBalances).map(item => ({
+    ...item,
+    usdValue: item.userBalance.usdValue as BigNumber,
+    asset: `_ceth_${item.address}`,
+    premiumOnly: true,
+    assets: item.tokens.map(asset => ({
+      ...asset,
+      asset: asset.token
+    })),
+    type: 'token'
   }));
+
+  return [
+    ...mappedUniswapV3Balances,
+    ...mappedUniswapV2Balances,
+    ...mappedSushiswapBalances,
+    ...mappedBalancerBalances
+  ]
+    .sort((a, b) => sortDesc(a.usdValue, b.usdValue))
+    .map((item, id) => ({ ...item, id }));
```

Comment:
Again here maybe this functionality could be added to a store instead of pilling it up to the component itself, or maybe into a composable somewhere if a store is overkill